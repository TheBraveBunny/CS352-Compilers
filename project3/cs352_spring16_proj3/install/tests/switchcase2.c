int thing;

int foo(int i)
{
   switch(i){
   	case 1:
		printf("1!\n");
		i=i+2;
	case 0:
	default: 
		printf("not 1!\n");
   }


   printf("a = %d", i);

   return 1;
}
main()
{
   foo(0);
}

