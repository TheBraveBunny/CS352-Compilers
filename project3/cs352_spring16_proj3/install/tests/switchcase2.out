#functions: 2
#global vars: 1
#local vars: 0
#statements: 7
